src/emitter.js

```diff
@@ -79,10 +75,6 @@ Emitter.prototype = {
    * @param {*} payload Value to send with callback
    */
   _emit (event, payload) {
-    if (this._events == null) {
-      return this
-    }
-
     let i = 0
     while (i < this._events.length) {
       var cb = this._events[i]
```

Comment:
By always having an array, V8 can optimize the event emitter way harder. The increase to memory usage is negligible.